openlibrary/plugins/worksearch/bulk_search.py

```diff
@@ -0,0 +1,17 @@
+import web
+
+from infogami import config
+from infogami.utils import delegate
+from infogami.utils.view import public, safeint, render
+
+
+class bulk_search(delegate.page):
+    path = "/search/bulk"
+
+    def GET(self):
+        i = web.input(q='')
+        return render['bulk_search/bulk_search'](q=i)
```

Comment:
No longer has param!

```suggestion
        return render['bulk_search/bulk_search']()
```